front/app/components/EventCard/EventInformation/index.tsx

```diff
@@ -100,7 +99,9 @@ const StyledIcon = styled(Icon)`
   `}
 `;
 
-const EventDescription = styled.div``;
+const EventDescription = styled.div`
+  margin-bottom: 24px;
```

Comment:
Fixes 
<img width="1512" alt="DONE - Margin event info and file attachments" src="https://user-images.githubusercontent.com/16427929/225950706-76f56dc2-32fe-4b7a-b14d-d0f72faaf6d1.png">